docs/content/Getting-Started/Cloud/03-Generate-models.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Generate models
+permalink: /getting-started/cloud/generate-models
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 4
+isDisableFeedbackBlock: true
+---
+
+## Select tables
+
+Start by selecting the database tables to generate the data schema from, then
+click **Measures and Dimensions**:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Generate Models Screen detected tables from data source for selection"
+    src="https://ucarecdn.com/e0420b86-64bc-4e01-aa74-16a7e926f523/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Measures and dimensions
+
+Next, check the detected measures and dimensions to ensure correctness. After
+confirming they are indeed correct, click **Primary Keys** to progress to the
+next step:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Generate Models Screen showing detected measures and dimensions"
+    src="https://ucarecdn.com/e0f8e8c3-9961-4360-ab9f-cfd96306c832/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Primary keys
+
+Review the detected primary keys for each cube and click **Joins** to move to
+the next step:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Generate Models Screen showing detected primary keys"
+    src="https://ucarecdn.com/e13e8279-e9e9-40df-b8f6-8cf5bb34ee14/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Joins
+
+Once again, review the detected joins and click **Review**:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Generate Models Screen showing detected joins"
+    src="https://ucarecdn.com/8190bd70-e4e3-4e9f-9cee-e9416ddf964b/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Review
+
+Finally, the last screen allows you to review all the detected cubes and their
+corresponding measures, dimensions and joins. If everything looks good, click
+**Confirm & Generate**:
```

Comment:
There's also an option to select data model language there: YAML or JavaScript. I think we need to explain the trade-off and provide a link to relevant sections.